worker-api/src/main/java/com/hpe/caf/api/worker/TaskMessage.java

```diff
@@ -87,6 +87,11 @@
      * Task message priority.
      */
     private Integer priority;
+    
+    /**
+     * This field contains the correlation id which is eventually logged via MDC.
+     */
+    private String correlationID;
```

Comment:
Addressed `correlationId` casing. 
`TASK_MESSAGE_VERSION` - I found these instances of where it is getting used http://16.103.38.246:9090/search?group=1_verity&group=2_caf&group=3_Other-FAS-Libraries&full=%22workertask.getVersion%22&defs=&refs=&path=&hist=&type=&xrd=&nn=3&si=full&si=full

In case, we want to reject any messages which do not have this change, we can reject those messages by incrementing `TASK_MESSAGE_VERSION`. For correlation id change, I don't see a need to increment the version. But not entirely sure about any other use-case.